tasks/dependencies.rb

```diff
@@ -78,6 +78,16 @@
     bundle "install", cwd: "acceptance", delete_gemfile_lock: !conservative
   end
 
+  def latest_gem_version(gem_name)
```

Comment:
Originally, but we're back to `gem list -r` because in-process gem stuff just goes wrong too often.